lib/normalize-option.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+var valueOrFunction = require('value-or-function');
+
+module.exports = function normalizeOption(types, defaultValue, value) {
```

Comment:
I like function composition more than a lot of arguments.  I prefer 1 or 2 arguments (3 max) but the applied arguments make the calls longer.